windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -104,6 +115,8 @@ Windows nodes are now fully integrated with most of the monitoring capabilities
 [id="wmco-2-0-0-known-issues"]
 == Known issues
 
+* When you create a Windows pod with a projected volume with a set `RunAsUserName` option, the Windows pod will be stuck in the `ContainerCreating` state. This issue is due to file permissions not being correctly handled in the projected volume.
```

Comment:
We have a workaround for this. Rather you should say something along the lines of 

>`Windows Pods with RunAsUserName set in it's `SecurityContext` and has a Projected Volume associated with it, the file ownership permissions for the projected entities are not honored` 

as mentioned in https://github.com/kubernetes/kubernetes/issues/102849